.github/workflows/test_generation.yaml

```diff
@@ -63,4 +63,4 @@ jobs:
         working-directory: ${{ env.LIBRARY_CHECKOUT_PATH }}
         run: |
           git add -N . # Needed if files are untracked
-          git diff
\ No newline at end of file
+          git diff
```

Comment:
future: Showing the diff seems like a good idea! If this is to get a sense of the meaningful changes, it may be worth using `--ignore-all-space --ignore-blank-lines` to reduce noise unrelated to substantive changes.

From https://link.springer.com/article/10.1007/s10664-019-09772-z, it may be worth experimenting with using the histogram algorithm as well, on the notion that it produces simpler diffs.